tests/system/apps/ipv6-healthcheck.json

```diff
@@ -0,0 +1,25 @@
+{
+  "id": "/test-ipv6",
+  "instances": 1,
+  "cmd": "nc -kl -p 8080 -e sh -c $'sleep 1; echo -e \"HTTP/1.1 200 OK\\r\\nContent-Length: 3\\r\\n\\r\\nHi\"'",
```

Comment:
What is this script doing?

Reply:
that's a super awesome netcat listening on port 8080. This took me the most time from the whole PR :D support for webservers with ipv6 is very poor in general so this ended up to be the easiest way to do so.

Do you have any idea how to clarify that for the readers?

Reply:
A comment at `def ipv6_healthcheck()`?

Reply:
done

Reply:
Ah. Now I get it :).